options.js

```diff
@@ -3,5 +3,6 @@ export default {
     iconClasses: null,
     iconSide: '',
     searchWhen: 'onDelay',
-    timeoutDelay: 1000
+    timeoutDelay: 1000,
+    showEmpty: true
```

Comment:
I feel like this should default to false as it isn't really a function that makes sense to use in most cases.

Reply:
woops, I meant to do that. I was just testing that it worked